pkg/controller/persistentvolume/types.go

```diff
@@ -26,6 +26,14 @@ import (
 )
 
 const (
+	// A PVClaim can request a quality of service tier by adding this annotation.  The value of the annotation
+	// is arbitrary.  The values are pre-defined by a cluster admin and known to users when requesting a QoS.
+	// For example tiers might be gold, silver, and tin and the admin configures what that means for each volume plugin that can provision a volume.
+	qosProvisioningKey = "volume.experimental.kubernetes.io/quality-of-service"
```

Comment:
The key itself was defined by Tim.  I have no preference for the constant name.

This whole annotation will go away in the full provisioning feature for Kube 1.2.